app/views/create/next-steps.html

```diff
@@ -89,6 +89,11 @@ <h1 ng-if="!pendingTasks(tasks()).length && erroredTasks(tasks()).length">Comple
               </div>
             </div>
 
+            <div ng-if="templateMessage.length">
+              <h2>Template message</h2>
+              <p>{{templateMessage}}</p>
```

Comment:
look at osc-image-summary.html  the div with class of resource-description, it preserves newlines and does a  linkify on the text